addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java

```diff
@@ -36,11 +38,13 @@
  *
  * @author Björn Brings - Initial contribution
  */
+@NonNullByDefault
 public class ValloxMVHandler extends BaseThingHandler {
 
     private final Logger logger = LoggerFactory.getLogger(ValloxMVHandler.class);
-    private ScheduledFuture<?> updateTasks;
-    private ValloxMVWebSocket valloxSendSocket;
+    private @Nullable ScheduledFuture<?> updateTasks;
+    private @Nullable ValloxMVWebSocket valloxSendSocket;
+    private @Nullable WebSocketClient webSocketClient;
```

Comment:
One final comment: I consider this not to be `@Nullable`.

Reply:
You're right. Changed it.